viper/modules/rats/jrat.py

```diff
@@ -158,13 +169,14 @@ def parse_config(raw_config, enckey):
     config_dict["EncryptionKey"] = enckey.encode('hex')
     return config_dict
 
+
 def config(data):
     enckey, conf = get_parts(data)
-    if enckey == None:
+    if not enckey:
```

Comment:
My point is that isn't the same ;) 

`if enckey is None` is the same as `if enckey == None`, when `if not enckey` is the same as `if enckey == 0 or len(enckey) == 0 or enckey is None`